pyproject.toml

```diff
@@ -11,10 +11,16 @@ authors = [
 [tool.poetry.dependencies]
 python = "~3.8"
 pre-commit = {version = "^3.2.0", source = "PyPI"}
-cvtoolkit = { git = "https://github.com/Computer-Vision-Team-Amsterdam/CVToolkit.git",  tag = "v1.0.2" }
 pydantic = {version = "^1.8.2", source = "PyPI"}
 azure-iot-device = {version = "^2.13.0", source = "PyPI"}
 azure-storage-blob = {version = "^12.19.0", source = "PyPI"}
+aml_interface = { git = "https://github.com/Computer-Vision-Team-Amsterdam/AML-Interface.git",  tag = "v1.0.0" }
+azure-ai-ml = {version = "^1.2.0", source = "PyPI"}
+cvtoolkit = { git = "https://github.com/Computer-Vision-Team-Amsterdam/CVToolkit.git",  tag = "v1.0.2" }
+mldesigner = {version = "^0.1.0-beta.9", source = "PyPI"}
+numpy = {version = "^1.24.2", source = "PyPI"}
```

Comment:
numpy and py360convert are not used in the new code, were they missing before? Otherwise can be removed here.